src/test/ui/consts/const-mut-refs/mut_ref_in_final_ok.rs

```diff
@@ -0,0 +1,22 @@
+#![feature(const_mut_refs)]
+#![feature(const_fn)]
+#![feature(raw_ref_op)]
+
+use std::cell::UnsafeCell;
+struct NotAMutex<T>(UnsafeCell<T>);
+
+unsafe impl<T> Sync for NotAMutex<T> {}
+
+const FOO: NotAMutex<&mut i32> = NotAMutex(UnsafeCell::new(&mut 42));
+//~^ ERROR temporary value dropped while borrowed
```

Comment:
Could you add the same test also for a `static` and a `static mut`?

Reply:
Also why is this a separate test from `mut_ref_in_final.rs`? The "ok" in the filename does not even make much sense, there is an ERROR here after all.

Reply:
Same, this comment seems to still be open.